packages/client/package.json

```diff
@@ -11,17 +11,23 @@
     }
   },
   "scripts": {
-    "build": "tsup src/index.ts --dts --format esm,cjs",
-    "clean": "rm -rf dist",
+    "build": "yarn generate-types && tsup src/index.ts --dts --format esm,cjs",
+    "dev": "yarn generate-types && tsup src/index.ts --watch --dts --format esm,cjs",
+    "clean": "rm -rf dist src/generated",
     "lint:fix": "yarn lint --fix",
-    "lint": "eslint"
+    "lint": "eslint",
+    "generate-types": "graphql-codegen --config codegen.yml"
   },
   "dependencies": {
     "graphql": "16.10.0",
     "graphql-request": "7.1.2",
     "graphql-tag": "2.12.6"
   },
   "devDependencies": {
+    "@graphql-codegen/cli": "5.0.5",
```

Comment:
I think we should add a dependency on the backend here so that `/../backend/generated/schema.graphql` get built first!